python/common/org/python/types/Str.java

```diff
@@ -1148,10 +1149,21 @@ private long toPositiveIndex(long index) {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "strip",
+            default_args = {"charsToBeRm"}
     )
-    public org.python.Object strip() {
-        throw new org.python.exceptions.NotImplementedError("strip() has not been implemented.");
+    public org.python.Object strip(org.python.Object charsToBeRm) {
+        String strim = "";
```

Comment:
As I mentioned in the previous pull request, this is best implemented by calling rstrip and lstrip, like:

```
    public org.python.Object strip(org.python.Object chars) {
        return ((Str ) this.rstrip(chars)).lstrip(chars);
    }
```